search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -198,75 +200,92 @@ class TemporalEntityAttributeService(
 
     fun injectTemporalValues(
```

Comment:
I'll review the changes in this method after other modifications are done

Reply:
done in 7b63c88

Reply:
This function will deserve a refactoring later (probably when we integrate the more general search capabilities) but it seems to work as expected